<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>701</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="ruleTable">
         <column>
          <property name="text">
           <string>No</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Src_ipv4</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Src_port</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Des_ipv4</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Des_port</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Protocol</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Run_State_Label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>555</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FireWall isStoppd</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_red">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>540</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image/Red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_green">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>540</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image/Green.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="RUN_BUTTON">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ADD_RULE_BUTTON">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD RUlE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DELETE_RULE_BUTTON">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DELETE RULE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="STOP_BUTTON">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RuleTable</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
